Disable unselected if-menu rules and conditions when saving the menu

rxnlabs opened the pull request:
- Disable the unselected if-menu rules for menu items and prevent them from being submitted when the menu is saved
- Decrease the number of input variables to decrease the likelihood that PHP will disregard if-menu rules for nav items in large menus

helgatheviking commented:
I think the 3rd commit is what really closes #1. I may have broken your JS enqueing as apparently you need the JS script everywhere to handle the notice dismissal.

AndreiIgna commented:
👍 will check this and include it, thanks for the PR

rxnlabs committed:
Disable unselected if-menu rules and conditions when saving the menu

- Disable the unselected if-menu rules for menu items and prevent them from being submitted when the menu is saved
- Decrease the number of input variables to decrease the likelihood that PHP will disregard if-menu rules for nav items in large menus
